Gruntfile.coffee

```diff
@@ -1329,6 +1345,15 @@ module.exports = (grunt) ->
 				options:
 					testname: "Local Test - <%= grunt.template.today('yyyy-mm-dd hh:MM') %>"
 
+		"string-replace":
+			inline:
+				files:
+					'dist/wet-boew/js/': 'dist/wet-boew/js/*.js'
+				options:
+					replacements: [
```

Comment:
Hello @lucas-hay . We found a quick workaround internally to get the issue sorted and I had to move onto other stuff. I am working on it right now to get it to a better point hopefully good enough to ship.